.ci/Jenkinsfile

```diff
@@ -188,7 +188,13 @@ def generateStep(version){
 }
 
 def releasePackages(){
-  withSecretVault(secret: 'secret/apm-team/ci/apm-agent-python-twine',
```

Comment:
What happen with this vault key, `secret/apm-team/ci/apm-agent-python-twine`? Must be updated/revoked?

Reply:
I'll delete that user on PyPI once we verified that the token auth works as intended. At that point, we can also delete it in the vault (or delete it now if that's easier, with the minimal risk that we'll have to re-add it if this doesn't work out)